roles/eth2client/tasks/start-client-metric-push.yml

```
@@ -10,4 +10,4 @@
     name: "metric-pusher"
     minute: "*"
     job: "/opt/metric-pusher.sh"
-    user: "ubuntu"
+    user: "devops"
```

```suggestion
    user: "{{ ansible_user }}"
```
If "devops" is the ansible user, let's use the var instead